### PR TITLE
Add docs getting started use a sparse model page

### PR DESCRIPTION
**Pull request by markurtz**

### CODE REVIEW

File: src/content/get-started/use-a-sparse-model.mdx [hunk 1]
```diff
@@ -1,9 +1,95 @@
 ---
 title: "Use a Sparse Model"
 metaTitle: "Use a Sparse Model"
-metaDescription: "Use a Sparse Model"
+metaDescription: "Use a Sparse Model with DeepSparse to deploy for faster and cheaper inference on CPUs"
```
Comment (jeanniefinks) [inline src/content/get-started/use-a-sparse-model.mdx:4]:
```suggestion
metaDescription: "Use a Sparse Model with DeepSparse Engine to deploy for faster and cheaper inference on CPUs"
```

File: src/content/get-started/use-a-sparse-model.mdx [hunk 2]
```diff
@@ -1,9 +1,95 @@
...
+
+## Benchmark
+
+After executing `pip install deepsparse`, the benchmark tool `deepsparse.benchmark` will be available on your command-line interface.
+
+With this command you can easily compare the performance between any ONNX or SparseZoo models.
+
+#### Sentiment Analysis with DistilBERT
+
+Here we'll be showing the performance gain on a DistilBERT trained on the SST2 dataset, going from dense to sparse to sparse quantized:
```
Comment (jeanniefinks) [inline src/content/get-started/use-a-sparse-model.mdx:25]:
```suggestion
Here's the performance gain on a DistilBERT trained on the SST2 dataset, going from dense to sparse to sparse quantized:
```

File: src/content/get-started/use-a-sparse-model.mdx [hunk 3]
```diff
@@ -1,9 +1,95 @@
...
+Throughput (items/sec): 203.2273
+Latency Mean (ms/batch): 9.8183
+Latency Median (ms/batch): 9.7477
+Latency Std (ms/batch): 0.4578
+Iterations: 2034
+```
+
+#### Object Detection with YOLOv5
+
+Here we'll be showing the performance gain on a YOLOv5s trained on the COCO dataset, going from dense to sparse to sparse quantized:
```
Comment (jeanniefinks) [inline src/content/get-started/use-a-sparse-model.mdx:62]:
```suggestion
Here's the performance gain on a YOLOv5s trained on the COCO dataset, going from dense to sparse to sparse quantized:
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
minor edits

File: src/content/get-started/use-a-sparse-model.mdx [hunk 4]
```diff
@@ -1,9 +1,30 @@
...
+
+[Sparsification techniques](../../user-guide/sparsification) enable the removal of redundant information from over-parameterized neural networks while retaining the original accuracy.
+These sparsified models are smaller and can be deployed for cheaper and faster inferences.
+The [DeepSparse Engine](../../products/deepsparse), specifically, leverages the sparsified models to reduce compute and accelerate CPU inference to GPU performance levels.
+
+The process of sparsifying models involves new concepts and adjustments to models and pipelines that may take some time to set up.
+For this reason and more, the [SparseZoo](../../products/sparsezoo) contains pre-sparsified models that are ready for testing and deployment in addition to the [sparsification recipes](../../user-guide/recipes) used to create them.
+
+**SparseZoo stubs** enable you to reference any model on the SparseZoo in a convenient and identifiable way.
+SparseZoo stubs are found throughout the documentation for the DeepSparse Platform in addition to the model pages on the [SparseZoo Website](https://sparsezoo.neuralmagic.com)
```
Comment (jeanniefinks) [inline src/content/get-started/use-a-sparse-model.mdx:19]:
```suggestion
SparseZoo stubs are found throughout the documentation for the DeepSparse Platform in addition to the model pages on the [SparseZoo Website](https://sparsezoo.neuralmagic.com).
```
```suggestion
SparseZoo stubs are found throughout the documentation for the DeepSparse Platform in addition to the model pages on the [SparseZoo Website](https://sparsezoo.neuralmagic.com)
```

File: src/content/get-started/use-a-sparse-model/object-detection.mdx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+This will automatically download the model from the SparseZoo to your local machine and then compile it with the DeepSparse Engine.
+Once compiled, the model pipeline is ready for inference with any image.
+
+To use the pipeline, first run the following to download the sample image used in this example:
+
+```bash
+wget -O basilica.jpg https://raw.githubusercontent.com/neuralmagic/deepsparse/main/src/deepsparse/yolo/sample_images/basilica.jpg
+```
+
+Next instantiate the pipeline and pass the image in using the images argument:
```
Comment (jeanniefinks) [inline src/content/get-started/use-a-sparse-model/object-detection.mdx:37]:
```suggestion
Next, instantiate the pipeline and pass the image in using the images argument:
```
```suggestion
Next instantiate the pipeline and pass the image in using the images argument:
```

**Review by jeanniefinks — CHANGES_REQUESTED** [review]
Minor edits needed

**Comment by markurtz** [comment]
@jeanniefinks changes landed

**Review by jeanniefinks — APPROVED** [review]

**Pull request merged**